### DISCUSSION TITLE
Signing for "gRPC-C++-gRPCCertificates-Cpp" requires a development team

### DISCUSSION THREAD

**Discussion by umer322** [codemagic-ci-cd/codemagic-docs#1355] [Q&A] [answered]
Hi i am trying to build ios app on codemagic and after xcode archive is done i get this error can someone let me know how to resolve it?

build id is "63d2fb64fdf5ec7f095d81b6"

`  Result bundle written to path:
    	/var/folders/m7/h1mg7c7x40ddjz6mxjxm3htr0000gn/T/flutter_tools.MM6iMh/flutter_ios_build_temp_dirLMtY4s/temporary_xcresult_bundle

Error (Xcode): Signing for "gRPC-C++-gRPCCertificates-Cpp" requires a development team. Select a development team in the Signing & Capabilities editor.
/Users/builder/clone/ios/Pods/Pods.xcodeproj

Error (Xcode): Signing for "square_in_app_payments-sqip_flutter_resource" requires a development team. Select a development team in the Signing & Capabilities editor.
/Users/builder/clone/ios/Pods/Pods.xcodeproj

Encountered error while archiving for device.`

**Comment by himesh-cm** [comment, marked as answer]
Hi @umer322, Thanks for sharing the info, Let me check and get back to you.

**Reply by himesh-cm** [reply]
@umer322, Sorry for not getting to you bit earlier, i see that you have managed to resolve the issue.
Can you please confirm, if there is anything else you need assistance with?

**Reply by umer322** [reply]
Yes i resolved the issue by doing changes in the podfile

**Reply by mabehiry** [reply]
Hello,
Can you share the changes in podfile? I had the same issue
